src/Core/src/Platform/iOS/EntryExtensions.cs

```diff
@@ -37,5 +37,13 @@ public static void UpdateIsPassword(this UITextField textField, IEntry entry)
 			else
 				textField.SecureTextEntry = entry.IsPassword;
 		}
+
+		public static void UpdateIsTextPredictionEnabled(this UITextField textField, IEntry entry) 
+		{
+			if (!entry.IsTextPredictionEnabled)
+			{
+				textField.AutocorrectionType = UITextAutocorrectionType.No;
```

Comment:
Is there a way to unset?

Reply:
To tests this, you can merge this branch into your branch (if we don't merge my PR first) and write a change test like this: https://github.com/dotnet/maui/pull/426/files#diff-5299bd70e5171c15f386cf349244c62764e6373fb1bbcbe0047e901e2b7ebe7cR60

Reply:
The last commit should fix this :)

Reply:
Awesome!